components/Nav/Navbar/MobileMenu/index.tsx

```diff
@@ -132,6 +132,20 @@ export default (({ account, className }) => {
                                             />
                                             Portfolio
                                         </div>
+                                        <div
+                                            className={classNames(
+                                                linkStyles,
+                                                route.startsWith('trading') ? selectedStyles : '',
+                                            )}
+                                            onClick={() => handleRoute('/trading-comp')}
+                                        >
+                                            <img
+                                                className="mr-2 inline"
+                                                src={'/img/general/trading-comp.svg'}
+                                                alt="Trading Comp"
+                                            />
+                                            Trading Comp
+                                        </div>
```

Comment:
do you mind quickly aligning this with the other links as well, remove the stake page and stuff